python/ray/autoscaler/_private/aws/config.py

```diff
@@ -529,7 +579,8 @@ def _get_or_create_vpc_security_groups(conf, node_types):
     node_type_to_vpc = {
         node_type: _get_vpc_id_or_die(
             ec2,
-            conf[NODE_KIND_CONFIG_KEYS[node_type]]["SubnetIds"][0],
+            conf["available_node_types"][node_type]["node_config"]["SubnetIds"]
+            [0],
```

Comment:
@allenyin55 @pcmoritz didn't we uncover an issue here during the last SubnetId incident? Is it safe to just use the VPC of the first subnet (are we guaranteed they are all going to be in the same vpc)?

Reply:
Fixed by https://github.com/ray-project/ray/pull/14868?

Reply:
Ah, yes, thank you :) Couldn't find whether we had fixed it or not